downstream/titles/onboarding/master.adoc

```diff
@@ -0,0 +1,151 @@
+
+Using Automation Service Catalog you can:
+
+* apply multi-level approval to individual platform inventories
+* organize content in the form of products from your platforms into portfolios
+* choose portfolios to share with specific groups of users
+* set boundaries around values driving execution of user requests
+
+
+
+== Onboarding to Automation Service Catalog
+
+This guide provides instructions to begin using Automation Service Catalog, including prerequisites and procedures for connecting your Ansible Tower provider, as well as configuring users and permissions. We also provide instructions for working with platforms, portfolios, and products.
+
+After completing the setup described and following the steps in this guide, you will have a portfolio of products accessible to users.
+
+
+== Entitlements
+
+To begin using Automation Service Catalog, you need:
+
+At least one Red Hat account user with Organization Administrator entitlements.
+The account number for this Red Hat corporate account.
+
+=== Adding entitlements for Automation Service Catalog
+
+
+To use Automation Service Catalog as a Lighthouse customer, create a new Red Hat user account:
+
+. Register a new Red Hat corporate user account here.  This will be the Organization Administrator account and will be separate from any current support accounts you may have.
+. Account type:  Corporate
+. Red Hat Login: [Create a new login separate from any existing support IDs you may have today - maybe include “lighthouse” or “beta” in the userID]
+. Email address: [This may be the same as an email address on an existing support ID.]
+. Password: [Just needs to meet minimum requirements]
+. Company: [Append “Lighthouse” to your company name to ensure this stays separate from any existing support contracts, for example - Acme Lighthouse]
+. Complete the rest of the form as usual.
+. Send your new login ID to customerservice@redhat.com to obtain an account number.
+
+[NOTE]
+====
+The account number for your Red Hat corporate account is required to use Automation Service Catalog, but  is not provided automatically to Red Hat staff when creating a new account.
+====
+
+The required entitlements are added and you can now access Automation Service Catalog.
+
+== Groups and users
+
+When configuring Automation Service Catalog for your needs, consider the scope of your user environments and how those users are grouped.
+
+Determine the required level of access for your users based on the following predefined roles in Automation Service Catalog:
+
+* *Organization Administrator*: a Red Hat account user who can manage users and control their access and permissions.
+* *Approval Administrator*: 	An Approval administrator role that grants create, read, update and destroy permissions.
+* *Approval Approver*: 	An Approval approver role that grants read and create permissions.
```

Comment:
Approval approver has read permission to requests, create permission to actions.

Reply:
hey @bzwei and @mkanoor if the Approval approver only has read access to requests I think we need to rename that role as in its present state it suggests the user can approve requests. So if we're creating groups for approval processes, the users in that group all need to be Approval administrators to approval orders? Am I understanding that correctly?

Reply:
@bzwei @mkanoor Approval approver does also have approve/deny permissions, right? That is the primary role for adding to an approval group?

Reply:
@mkanoor directed me to the UX effort for these topics, working with them on providing concise descriptions per role.

Reply:
@cbudz It might make sense to track this file
https://github.com/RedHatInsights/rbac-config/blob/master/configs/approval.json
and this one for catalog
https://github.com/RedHatInsights/rbac-config/blob/master/configs/catalog.json

It has all the permissions for each of the roles so we can match it with the docs.